<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageManuallyClassifier</class>
 <widget class="QWidget" name="imageManuallyClassifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Manually Classifier Form</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cboClassName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblSegmentImage">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>501</width>
     <height>471</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Mode</string>
   </property>
   <widget class="QRadioButton" name="radMarkingMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Marking</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radRepairMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Repair</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
